docs/data/charts/tooltip/tooltip.md

```diff
@@ -90,6 +90,24 @@ In this demo, you can see:
 
 {{"demo": "AxisFormatter.js"}}
 
+### Label formatting
+
+The label text inside the tooltip can also be formatted conditionally by providing a function to the series' `label` property.
+
+```jsx
+<LineChart
+  // ...
+  series={[
+    {
+      data: [ ... ],
+      label: (location) => location === 'tooltip' ? 'BR' : 'Brazil'
+    }
+  ]}
+/>
+```
+
+To know more, head out to the [label page](/x/react-charts/label/#conditional-formatting)
```

Comment:
To align with: https://mui.com/x/react-date-pickers/date-field/#customize-the-date-format
```suggestion
:::info
See [Label—Conditional formatting](/x/react-charts/label/#conditional-formatting) for more details.
:::
```